### ISSUE TITLE
Path problems in Operation Inner Space + Hang

### ISSUE THREAD

**Issue by dreimer1986** [otya128/winevdm#3]
The game only "works" when you copy all files directly to C:\<root>. Otherwise it does not find it's files regardless what folder is the true root of the game and is set up in the ini file. I tried to cd into the real root, drag+drop and moving winevdm into the game root and retry both ways before. No dice.

--

https://pastiebin.com/5b4754c0d7fd0 Debug Log of the game starting and dying

**Comment by otya128** [comment]
What error messages is displayed?

**Comment by dreimer1986** [comment]
On the game? Well it says that it cannot find the ini file in plain english. When it works with my "workaround" no error shows up. the game just errors out in the typical Windows manner (ossc stopped working) and this paste is what the console shows me then.

**Comment by otya128** [comment]
I tried Operation Inner Space V1.29.07(demo).
It works in the latest version. 
So please try the latest version.
https://github.com/otya128/winevdm/releases/tag/v0.3.0
